source/tablist.rst

```diff
@@ -0,0 +1,9 @@
+==================
+PlayerList/Tablist
+==================
+
+Adventure does currently only support changing the header and footer of the tablist through
+``Audience#sendPlayerListHeader``, ``Audience#sendPlayerListFooter`` and ``Audience#sendPlayerListHeaderAndFooter``.
+
+Sending only a header might also display an existing footer. Likewise, sending only a footer might also display
+an existing header. This can vary depending on the platform you are running on.
```

Comment:
Maybe mention that it's more likely to support keeping the same header when on a server, and proxies are more likely to only let you set everything at once?

Reply:
Im not familiar with these standards. Do they only apply to the header? Or is it in general with single parts vs both at once.

Reply:
It's single parts vs both at once. The packet MC uses can only set both at once, so servers have to store the last values in order to be able to only update one at a time.